lovr-mouse.lua

```diff
@@ -1,4 +1,6 @@
-local ffi = require 'ffi'
+local ffi = assert(type(jit) == 'table' and               -- Only run if we have LuaJIT
+  lovr.getOS() ~= 'Android' and lovr.getOS() ~= 'Web' and -- and also GLFW
+  require 'ffi', "lovr-mouse cannot run on this platform")
```

Comment:
I find this expression a little hard to read because of the way things are split up (Sorry, I was the one who sketched it out all terse earlier in Slack).  Maybe semantically named variables could be used to break things up a bit and reduce the need for the comments?:

```lua
local luajit = type(jit) == 'table'
local desktop = lovr.getOS() ~= 'Android' and lovr.getOS() ~= 'Web'
local ffi = assert(luajit and desktop and require 'ffi', '')

-- or assert could be separated from require:
-- assert(luajit and desktop, '')
-- local ffi = require 'ffi'
```

The following would help me read it better too, but is a still harder to parse than the above IMO:

```lua
local ffi = assert(
  type(jit) == 'table' and
  lovr.getOS() ~= 'Android' and
  lovr.getOS() ~= 'Web' and
  require 'ffi',
  'no'
)
```